### ISSUE TITLE
Fires and bombs are capable of destroying floors in away missions. This is bad.

### ISSUE THREAD

**Issue by WJohn** [NTStation/NTstation13#826]
Currently, if you start a large and very hot fire in away missions, it can melt through the floor, exposing everything to space and draining the atmosphere. This is extremely bad for planet-side away missions as it can drain all the air, cause lots of lags, and break immulsions.
The easy fix for this would be to cap fires in away missions at 9,800K degrees so it can never melt any floor.

At the same time bombs are also able to destroy the floor, exposing it to space and draining all the air. I have never tried to put C4 on floor, but I assume it'll delete it and expose it to space, too.
The easy fix for this would be to prevent all explosions from doing any heavy or total devastation damage when in an away mission (light damage and flash are fine, and the fires are too brief to do any damage), and to prevent C4 from being able to delete floors when in away missions.

**Comment by Atskadan** [comment]
Aren't there indestructible floors?

**Comment by WJohn** [comment]
Yes, those exist but I refuse to use them as they are boring and unresponsive to your interactions. They can't simulate air, you can't repair them if they look broken, you can't replace them or do anything. It's overall very rigid.
